projects/1.16/assets/tinkers-construct/tconstruct/book/mighty_smelting/zh_cn/abilities/harvest.json

```
@@ -0,0 +1,12 @@
+{
+  "title": "收集能力",
+  "text": [
+    {
+      "text": "轻松收获作物！额收集好像不单指收获作物……这些升级能使你在某些情境下提高收集各式方块的效率！"
```

```suggestion
      "text": "轻松收获作物！不过收集好像不单指收获作物……这些升级能使你在某些情境下提高收集各式方块的效率！"
```
还是严防书面文字口语化吧。该写出来的东西我觉得不要看着像是说出来的。